content/developers/api-reference/events-api/index.md

```diff
@@ -391,6 +391,90 @@ You should see the response:
 }
 ```
 
+### Event Record Retrieval
+
+Event records in DataTrails are tokenized at creation time and referred to in all API calls and smart contracts throughout the system by a unique identity of the form:
+
+```bash
+assets/xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx/events/xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
+```
+
+If you do not know the Event’s identity you can fetch Event records using other information you do know.
+
+#### Fetch All Events
+
+To fetch all Event records, simply `GET` the Events resources:
+
+```bash
+curl -v -X GET \
+     -H "@$HOME/.datatrails/bearer-token.txt" \
+     "https://app.datatrails.ai/archivist/v2/assets/-/events"
+```
+
+#### Fetch Events for a Specific Asset
+
+If you know the unique identity of the Asset record simply `GET` the resource:
+
+```bash
+curl -v -X GET \
+     -H "@$HOME/.datatrails/bearer-token.txt" \
+     "https://app.datatrails.ai/archivist/v2/assets/xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx/events"
+```
+
+#### Fetch Specific Events by Identity
```

Comment:
This should be at the top before you make the note "If you don't know the identity ... " - or at least it should be the very first example after it - see what flows better.

Reply:
Hi @JAG-UK  may be confused as there is a note before this example that states . . . 

"If you know the unique identity of the Asset record . . . " as it represents retrieving events for a specific Asset thus all a person would require to know is the Asset ID.

Then following is if you know the the unique ID for the Asset and Events. . . 

Was following the flow of the Asset page. . . seems in step with what we have documented.  Are you stating that we should have a line that states "If you don't know the identity  . . ."? seems redundant as it's the other side of the coin of just knowing the Asset ID   . . . retrieving events for a specific Asset.